README.md

```diff
@@ -8,8 +8,7 @@ your git repositories.
 ## Install
 
 ```
-$ curl https://raw.githubusercontent.com/joshfriend/hooker/master/hooker.sh
-$ ./hooker.sh --install
+$ bash <(curl -s https://raw.githubusercontent.com/joshfriend/hooker/master/hooker.sh) --install
```

Comment:
Oh, I didn't realize I had to add hooker.sh to my projects. I like the idea of hooker installing itself in the hooks directory.